docs/arcaflow/workflows/expressions.md

```diff
@@ -39,7 +41,51 @@ For example, to have the following text represented in a single string:
 
 You would need the expression `"test\ntest2/\\"`
 
-### Integer
+#### String-expressions in YAML
+
+If an expression has a string literal in it, then you must account for it in the YAML.
+
+For example, to include a double-quoted string in an expression, you must either add single quotes around expression, or use block flow scalars. A single apostrophe within an expression needs to be added twice in a row to count as one while in an inlined string.
```

Comment:
And, there's another one of those commas preceding a sentence fragment.  (If you want to keep the comma, you need to follow it with another "you must" -- otherwise you have no subject in the second clause.)

The second sentence is missing context:  the doubling requirement is only true inside a single-quoted string, right?  It's not (necessarily) true in a block flow.  I would add "Inside a single-quoted string" before "a single apostrophe".